test/modules/ModuleLib/ModuleUtil.cs

```diff
@@ -120,5 +122,17 @@ static void WriteLog(ILogger logger, LogLevel logLevel, string message)
                 logger.Log(logLevel, message);
             }
         }
+
+        public static SortedDictionary<string, string> ParseTestInfo(string testInfo)
```

Comment:
nit: put a line of comment to specify the format of test Info.

Reply:
I was thinking of putting the specific fields, but decided against it because the comments can easily become outdated. I am going to put a generic description specifying where the test information comes from.

Reply:
please mention the format: key=value[,key=value]